### ISSUE TITLE
Top-level :time-zone changes system time zone

### ISSUE THREAD

**Issue by terjesb** [troy-west/cronut#5]
Thanks for cronut and its ideas. It facilitated moving some stuff off of WildFly and immutant.scheduling, and into a simpler Jetty setup. I was surprised to see that cronut changed the JVM default time zone when using a top-level :time-zone on the schedule, though. After digging through the code, I think the README could be clearer about how time zones work in cronut. At least I was bitten by this.

Please consider calling out in the README that using ``:cronut/scheduler {:time-zone ...}`` is not isolated to Quartz/cronut, but will actually change the JVM default time zone.

Thankfully, cronut also supports using :time-zone inside the job definition, however this only seems to work for #cronut/trigger and not #cronut/cron.

To avoid messing with the system time zone but still be able to specify a time zone for a job (and incidentally also be able to use different time zones for different cron jobs), this seems to work:

* do not use the top-level :time-zone (since it currently changes system default zone)
* do not use #cronut/cron (since it only supports a simple cron expression and not a job-specific timezone)
* instead, configure jobs like:

```
{:job ...
 :trigger #cronut/trigger {:type :cron
                           :cron "....."
                           :time-zone "Europe/Oslo"}}
```

This already works fine, maybe it could also be worked into the README?

A possible enhancement here could be to make the top-level :time-zone NOT change the default system zone (or non-breaking, by growing a top-level :default-time-zone), but instead act as the default :time-zone for each of the job definitions (and support it for both #cronut/cron and #cronut/trigger shortcuts), unless overridden to a different time-zone or nil for system time zone. This is how I believed top-level :time-zone would work in the first place.

Btw, do the data readers really add anything here? Maybe always using a map would work just as fine. ``:trigger #cronut/cron`` or ``:trigger #cronut/trigger`` doesn't buy that much over simply ``:cronut/trigger {:type :cron}``, it actually kinda buys less, but with a higher cost in multiple concepts and understanding?

**Comment by d-t-w** [comment]
Hello @terjesb, thank-you for this, I'm just digesting it now!

**Comment by d-t-w** [comment]
Thanks again @terjesb, I thought about this over the weekend and decided to remove the :time-zone configuration at the scheduler level. I agree Cronut shouldn't alter the system default time-zone, that was just my laziness for the particular problem I was solving at the time.

It's a breaking change, have released 0.2.0 and added a little to the README. I've left the shortcut tagged literals for cron and interval in, they're really useful in my case where I want a single time-zone to apply for all cron triggers and I only want to define the cron expression itself.

I experimented with using a scheduler-level time-zone to alter triggers after the fact (see the PR above), but the implementation relies on reflection and ultimately diverges from Quartz itself (there's no way to configure per-scheduler time-zones in Quartz, it just falls back to system default tz where none is set per trigger). I don't want to maintain an external library to Quartz, just wrap it in as data-oriented a manner as possible.

As a side note, my implementation would never have worked for multiple schedulers in a single JVM, which is something I'd like to support (via a pluggable SchedulerFactory) at a later time, so makes sense to pull this out now.
